FiveCalls/FiveCalls/IssueDone.swift

```diff
@@ -9,13 +9,17 @@
 import SwiftUI
 
 struct IssueDone: View {
+    @EnvironmentObject var router: Router
+    
     let issue: Issue
     
     var body: some View {
         ScrollView {
             VStack {
                 Text("Issue \(issue.name) done page")
-                Button(action: {}, label: {
+                Button(action: {
+                    router.backToRoot()
```

Comment:
nit: this could be done via the unidirectional store instead if we wanted to keep everything in that flow. We could add an action `backToRoot` (handled in middleware) that could be sent from anywhere that has access to the store without having to expose the routing details.

Reply:
I'm going to give this a shot to see how it feels but I'll do it in another PR